templates/core/Cargo.toml.example

```diff
@@ -42,3 +42,8 @@ rustc-args = ["--example-rustc-arg"]
 
 # Additional `RUSTDOCFLAGS` to set (default: [])
 rustdoc-args = ["--example-rustdoc-arg"]
+
+# List of command line arguments for `cargo`.
+#
+# These cannot be a subcommand, they may only be options.
```

Comment:
Maybe mention that these will be passed to multiple subcommands so need to be generally applicable.

Reply:
Is that true? I thought we only ever use the `rustdoc` subcommand.

I can add a disclaimer that we don't guarantee the subcommand and you should make these independent, but I don't know how useful that is in practice - they can't *truly* be independent because e.g. `cargo new` won't have any useful options, and I think almost everything else is shared between `rustdoc` and `doc`.

Reply:
We use `cargo metadata` too I think.

Reply:
Yes, but we don't pass user-defined args there: https://github.com/rust-lang/docs.rs/blob/20aa7060349175424e915368790a5f064bfc4ad7/src/utils/cargo_metadata.rs#L17-L21

Reply:
If it helps, the only place we call `cargo_args()` is for `prepare_command`: https://github.com/rust-lang/docs.rs/blob/20aa7060349175424e915368790a5f064bfc4ad7/src/docbuilder/rustwide_builder.rs#L588
which is only used for doc coverage and actually generating the documentation.